docs/gen2_Tutorial.md

```diff
@@ -429,6 +429,73 @@ and clean them up.
 
 If the cluster which which is being deleted is a cluster created using patch files, the controller first cleans the git repo where the respective patch files of the cluster are present and then it destroys all the related ArgoCD applications and clean them up.
 
+## Enabling Cluster Autoscaler in the workload cluster
+To create a gen2 cluster with autoscaler, we need:
+
+- bundle pointing to the bundle/capi-cluster-autoscaler in the arlon repository.
+- dynamic profile that contains the above bundle.
+- a basecluster manifest which has the CAPI annotations for min and max nodes set ( as a part of `preparegit` or manually add it ).
```

Comment:
The base cluster also needs to use MachineDeployments and not MachinePools.

Reply:
I will mention that part